.editorconfig

```diff
@@ -63,6 +63,18 @@ end_of_line = lf
 [Makefile]
 indent_style = tab
 
+##########################################
+# File Header (Uncomment to support file headers)
+# https://docs.microsoft.com/en-us/visualstudio/ide/reference/add-file-header
+##########################################
+
+# [*.{cs,csx,cake,vb,vbx}]
+# file_header_template = <copyright file="{fileName}" company="PROJECT-AUTHOR">\nCopyright (©) 2020 PROJECT-AUTHOR. All Rights Reserved\n</copyright>

```

Comment:
Please remove the explicit year from the rule.

Reply:
Removed